### DISCUSSION TITLE
I got a bladabindi win32 backdoor from this

### DISCUSSION THREAD

**Discussion by wayden-kok** [seevik2580/tor-ip-changer#97] [Q&A]
Can anyone tell me if you have the problem too?
and to the creator: are you able to explain this?
btw it was from the updater.exe

**Comment by seevik2580** [comment]
updater.exe is packed with https://nuitka.net/ without further inspection. its false positive because updater will download other stuff such as new version of ipchanger into your pc. and as a python downloader its marked as backdoor so if you dont believe my packed exe then you can always pack your own exe from source code with another packer.
